extensions/mailer/runtime/src/main/java/io/quarkus/mailer/runtime/BlockingMailerImpl.java

```diff
@@ -18,6 +19,10 @@
 
     @Override
     public void send(Mail... mails) {
+        if (!BlockingOperationControl.isBlockingAllowed()) {
+            throw new RuntimeException(
+                    "Attempted a blocking operation from the IO thread. If you want to send main from an IO thread please use ReactiveMailer instead, or dispatch to a worker thread to use the blocking mailer.");
```

Comment:
There's a typo in the error message:
> If you want to send main from...

Should have been "If you want to send mail from..."